force-app/main/default/classes/LeadService.cls

```diff
@@ -1,3 +1,9 @@
 public with sharing class LeadService {
-
+public static void UpdTriggerLeadStatus (list<Lead>newLead){
```

Comment:
@GulzatB method name should start with lowerCase, I think updateLeadStatus is better name.

Reply:
also please add Apex Doc above the method, you can check how it's written in ContactService.
add following annotations

@param
@author

Reply:
Yeah, sounds good :)